package.json

```diff
@@ -5,8 +5,11 @@
   "main": "dist/cli.js",
   "type": "module",
   "scripts": {
+    "postinstall": "npm run build",
     "build": "tsc && chmod +x dist/cli.js",
-    "watch": "tsc --watch"
+    "watch": "tsc --watch",
+    "format": "eslint --fix --ext .ts,.js src",
+    "generate": "VERBOSE=1 DEBUG=1 graphql-codegen --config codegen.yml"
```

Comment:
Did you get graphql-codegen working? If not we can probably rm all the graphql-codegen stuff